src/common/executor/ExecutorClient.js

```diff
@@ -376,6 +374,12 @@ define(['superagent', 'q'], function (superagent, Q) {
         if (this.executorNonce) {
             req.set('x-executor-nonce', this.executorNonce);
         }
+        if (this.apiToken) {
+            req.set('x-api-token', this.apiToken);
```

Comment:
I could change this to `x-access-token` if that is preferred.

Reply:
x-api-token is fine